Client/Frontend/DevicePickerTableViewCell.swift

```diff
@@ -0,0 +1,55 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+class DevicePickerTableViewCell: UITableViewCell {
+
+    private struct UX {
+        static let deviceRowTextFont = UIFont.systemFont(ofSize: 16)
+        static let deviceRowTextPaddingLeft: CGFloat = 72
+        static let deviceRowTextPaddingRight: CGFloat = 50
+    }
+
+    static let cellIdentifier = "ClientPickerTableViewCell"
```

Comment:
nit: Could we use @polymathruby's cell identifier extension? 🤔 

Look for `ReusableCell`